apps/server/src/modules/deletion/repo/mapper/deletion-log.mapper.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+import { deletionLogEntityFactory } from '../../entity/testing/factory/deletion-log.entity.factory';
+import { DeletionLogMapper } from './deletion-log.mapper';
+import { DeletionLog } from '../../domain/deletion-log.do';
+import { deletionLogFactory } from '../../domain/testing/factory/deletion-log.factory';
+import { DeletionLogEntity } from '../../entity';
+
+describe(DeletionLogMapper.name, () => {
+	describe('mapToDO', () => {
+		describe('When entity is mapped for domainObject', () => {
+			it('should properly map the entity to the domain object', () => {
+				const entity = deletionLogEntityFactory.build();
+
+				const domainObject = DeletionLogMapper.mapToDO(entity);
+
+				const expectedDomainObject = new DeletionLog({
+					id: entity.id,
+					domain: entity.domain,
+					operation: entity.operation,
+					deletionRequestId: entity.deletionRequestId?.toHexString(),
+					modifiedCount: entity.modifiedCount,
+					deletedCount: entity.deletedCount,
+					createdAt: entity.createdAt,
+					updatedAt: entity.updatedAt,
+				});
```

Comment:
Please move it (as well as the entity preparation in the first place) to the `setup()` function.

Reply:
done